### PR TITLE
Add Mercado Libre to portal adopters

### PR DESCRIPTION
**Pull request by privero-meli**

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -5,10 +5,13 @@ These adopters use the project either directly or as a reference for their own i
 
 | Organization | Details                                                                   | References                                                |
 | ------------ | ------------------------------------------------------------------------- | --------------------------------------------------------- |
+| [Mercado Libre][Meli]   | Mercado Libre is using this portal internally for discovery of InnerSource Projects | See the [known instances][known_instances] section of the Portal Pattern |
```
Comment (spier) [inline ADOPTERS.md:8]:
```suggestion
| [Mercado Libre][Meli]   | Mercado Libre is using this portal internally for discovery of InnerSource Projects | See the [known instances][known_instances] section of the InnerSource Portal pattern. |
```

Comment (spier) [reply ADOPTERS.md:8]: I would spell out the full name of the pattern.

Even thinking if maybe the link would be better to place on the pattern name, than on the "known instances".
While the terminology "known instances" has a specific mean for the InnerSource Commons patterns, it is probably not that easy to understand for the users of the SAP portal. Even leaving that phrase away entirely could be an option?

**Review by spier — COMMENTED** [review]
Great to see you contributing your adoption of the portal!
Thanks for sharing this.

File: ADOPTERS.md (hunk 1 above)
Comment (privero-meli) [reply ADOPTERS.md:8]: Naming the section helps to find the mention on the pattern page.

How about something like this?

`See the "known instances" section of the [InnerSource portal pattern](https://patterns.innersourcecommons.org/p/innersource-portal)

**Review by Michadelic — APPROVED** [review]
Thank you very much for listing your company in the adopters section.
It's great to hear that you find our project helpful

**Comment by Michadelic** [comment]
@privero-meli Please don't forget to sign the CLA agreement so that we can merge your contribution, thx 👍

**Pull request merged**